scip.proto

```diff
@@ -309,6 +320,13 @@ message SymbolInformation {
   // - The symbol may encode names with special characters that should not be
   //   displayed to the user.
   string display_name = 6;
+  // (optional) The signature of this symbol as it's displayed in API
+  // documentation or in hover tooltips. For example, a Java method that adds
+  // two numbers this would have `Document.language = "java"` and `Document.text
+  // = "void add(int a, int b)". The `language` and `text` fields are required
+  // while other fields such as `Documentation.occurrences` can be optionally
+  // included to support hyperlinking referenced symbols in the signature.
+  repeated Document signature_documentation = 7;
```

Comment:
Could you include an example of when it'd be useful for this field to have more than one element?

Reply:
Good point, we can make it non-repeated. It's the same binary encoding either way so we can make it repeated in the future if there is demand for it.

Reply:
Pushed a commit making it non-repeated